helmfile.d/config/kusama/otv-backend.yaml.gotmpl

```diff
@@ -1759,6 +1759,56 @@ config: |
               "name": "IBIZA-sv-validator-0",
               "stash": "E5sYCKxrZT4oJfiQxKvLZeK6HkYdo9Ettz1MFq3XgiGWJiH",
               "riotHandle": "@ibiza1:matrix.org"
+          },
+          {
+              "name": "ryanhigs",
```

Comment:
hey @wpank, I'm taking a look at the validator name here. Are there any issues with the `name` var not being the exact name of the subaccount?
ie. ryanhigs is a parent account, with ryanhigs/1 as a subaccount linked to `stash` `HLsh79QY2bnA5URc9jrxwwH6xvnqbzU59HDpgrJNgwLsdct`. if ryanhigs/2 comes along and is added to this list (separate stash), how might it impact this first item, if at all?

Reply:
These names are the telemetry names, not the names of the on chain identity

Reply:
Got it, thanks!